frontend/src/pages/Home.js

```diff
@@ -77,13 +89,18 @@ function getTabIndex(path) {
 
 export default function Home() {
   const [value, setValue] = React.useState(0);
-  const selectedNetwork = useSelector(state => state.common.selectedNetwork?.chainName || "")
+  const selectedNetwork = useSelector(
+    (state) => state.common.selectedNetwork?.chainName || ""
+  );
   const [network, setNetwork] = React.useState(selectedNetwork);
+  const networks = useSelector((state) => state.wallet.networks);
+  const wallet = useSelector((state) => state.wallet);
 
   const dispatch = useDispatch();
   const navigate = useNavigate();
   const location = useLocation();
-  const page = location.pathname.split('/')?.[location.pathname.split('/')?.length - 1]
+  const page =
+    location.pathname.split("/")?.[location.pathname.split("/")?.length - 1];
```

Comment:
Write as separate utility function if using in different files, if not define `location.pathname.split("/")` in separate variable and use it.